tic1.js

```diff
@@ -0,0 +1,127 @@
+// this is a test
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+      readlineInterface.question(questionText, resolve);
+    });
+  }
+
+let cell1 = 1
+let cell2 = 2
+let cell3 = 3
+let cell4 = 4
+let cell5 = 5
+let cell6 = 6
+let cell7 = 7
+let cell8 = 8
+let cell9 = 9
+let cells = [cell1, cell2, cell3, cell4, cell5, cell6, cell7, cell8, cell9]

```

Comment:
You've decided to use the cell number (or index) as a value for empty cells. That's workable but it does muddy the waters, mixing up the *data model* ("empty cell") with the *presentation* ("grid position 5"). Consider using only 3 values in each cell (e.g. 'X', 'O', or null) and then making the `printGrid` function responsible for figuring out what to display based on the value.